packages/blade/src/components/Tooltip/types.ts

```diff
@@ -1,8 +1,10 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import type { Side, UseFloatingOptions } from '@floating-ui/react';
```

Comment:
Very weird, it's a type import but still resolving the module. type imports are supposed to be stripped out from runtime. Not sure why this is occurring.

Reply:
I can see `@floating-ui/react` import on native bundle only coming it for `useFloatingOptions` so this is the only place where it could come from. I'll send Shubhnik this PR to try out before we conclude